.travis.yml

```
@@ -118,6 +118,11 @@ jobs:
       env: >-
         FILE_ENV="./ci/test/00_setup_env_i686.sh"
 
+    - stage: test
+      name: 'x86_64 Linux  [GOAL: install]  [CentOS 7]  [no depends, only system libs]'
+      env: >-
+        FILE_ENV="./ci/test/00_setup_env_centos.sh"
```

nit: This should run on any architecture 

```suggestion
        FILE_ENV="./ci/test/00_setup_env_native_centos.sh"
```

---

Done.